API.md

```diff
@@ -0,0 +1,114 @@
+# CACHE B2B API Documentation
+
+CACHE intends to make available API's for B2B customer's to build infrastructure experiences.
+CACHE mainly allows for the allocation of Gold Bullion. The API's are split into this core functionality and further
+supportive API's used for building B2C applications are also available under the CACHE Whitelabel category. 
+
+## Base URL
+We will open up the base url to B2B customer's. Interested developers can reach out to ```contact@cache.gold```
+For a quick reference please see ```https://api-prod.gramchain.net/api-docs/``` do note that it may not contain all the api's listed here.
+
+## Open Endpoints - Explorer API's and Authentication
+
+Open endpoints require no Authentication.
+
+* [Explore CACHE Holdings](https://api-prod.gramchain.net/api-docs/) : `POST  <baseurl>/api/public/GetTreeByGroup`
+* [Login](https://api-prod.gramchain.net/identity/login) : `POST <baseurl>/identity/login?signin=`
+
+
+## Endpoints that require Authentication
+
+Permissioned endpoints require a valid Token to be included in the header of the
+request. A Token can be acquired from the Login view above.
+
+### Bar Allocation related
+
+Endpoint allows for the deposit, reservation, redemption, proof of reserve verification(smart contract), fees accrued etc:
+
+* [Deposit]() : `GET /api/Secured/GramChainInsertViaWeb`
+* [Reserve a particular bar]() : ``
+* [Remove the reservation a particular bar]() : ``
+* [Verify the reservation status of bars by address]() : ``
+* [Mint new tokens]() : ``
```

Comment:
As discussed -- we should not show the capability to mint tokens (this is CACHE's job, not the third-party dev's)